64.md

```diff
@@ -0,0 +1,41 @@
+NIP-64
+======
+
+Trust Rank
+----------
+
+`draft` `optional`
+
+A trust rank is a public way to individually assign a quantifiable level of trust on specifc topics to a nostr identity.
+Ranks can be used to discover trusted people or content on a common topic.
+
+## Event Structure
+
+The `kind:30382` is used to assign public relationship statuses between the event author and a pubkey set to the `d` tag.
+This NIP assigns one or more trust ranks by adding `T` tags with different topic names to such event
+using the following structure: `["T", "<int_value_ranging_from_-3_to_3>:<topic-name>", "<context_dependent_marker>"]`.
```

Comment:
What do you think about `["T", "<topic-name>", "<int_value_ranging_from_-3_to_3>", "<context_dependent_marker>"]`?
This way you could easily query a pubkey's trust rank for a topic.

Reply:
Your example doesn't allow to filter on relays by trust score.

The reason the score has few possible values (-3 to 3) and goes together with the topic name is that it enables filtering on relays by both (always picking one or more specific topics).

Filter examples:
All "nostr": `{ #T: ['-3:wiki:nostr', '-2:wiki:nostr', "...", '3:wiki:nostr'] }`
Positive "nostr": `{ #T: ['1:wiki:nostr', '2:wiki:nostr', '3:wiki:nostr'] }`
Negative "nostr": `{ #T: ['-3:wiki:nostr', '-2:wiki:nostr', '-1:wiki:nostr'] }`